### ISSUE TITLE
[Accessibility] Three-dot menu icon has insufficient contrast

### ISSUE THREAD

**Issue by amkcpu** [FredJul/Flym#712]
One of the results of an accessibility scan with the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor) is that the three-dot menu icon in the main screen has insufficient contrast:

<img src="https://user-images.githubusercontent.com/46764284/97502954-71bc5200-1974-11eb-9463-07427eadc294.png" width=300 />

If you compare the three-dot icon with the search icon, it is a little darker. This should be changed to be plain white.

**Comment by amkcpu** [comment]
The same goes for the three-dot icon in articles (`EntryDetailsView`).

**Comment by andreacantone** [comment]
Even the title of the feeds does not respect the contrast ratio.
